dev/tools/dartdoc.dart

```diff
@@ -52,7 +53,15 @@ dependencies:
   if (code != 0)
     exit(code);
 
-  // Generate the documentation; we require dartdoc >= 0.9.4.
+  // Install dartdoc.
```

Comment:
Maybe a note that reminds us to remove this when the bug is fixed, and link to the bug?